pro/src/components/pages/Home/Offerers/OffererDetails.jsx

```diff
@@ -272,7 +279,38 @@ const OffererDetails = ({
                 équipes.
               </Banner>
             )}
-            {isUserOffererValidated && (
+            {!selectedOfferer.isValidated &&
+              selectedOfferer.userHasAccess &&
+              !hasAtLeastOnePhysicalVenue && (
+                <Banner
+                  type="notification-info"
+                  className="banner"
+                  links={[
+                    {
+                      href: `https://aide.passculture.app/hc/fr/articles/4514252662172--Acteurs-Culturels-S-inscrire-et-comprendre-le-fonctionnement-du-pass-Culture-cr%C3%A9ation-d-offres-gestion-des-r%C3%A9servations-remboursements-etc-`,
+                      linkTitle: 'En savoir plus',
+                      Icon: ExternalSiteIcon,
+                    },
+                  ]}
+                >
+                  <strong>Votre structure est en cours de validation</strong>
+                  <br />
+                  Nous vous invitons à créer un lieu afin de pouvoir proposer
+                  des offres physiques ou des événements. <br />
+                  Vous pouvez dès à présent créer des offres numériques.
+                  L’ensemble de ces offres seront publiées une fois votre
+                  structure validée.
```

Comment:
Cet accord au pluriel me pique un peu les yeux. J'aurais écrit "L'ensemble de ces offres sera publié". Après une rapide recherche, il semble que les 2 soient acceptés.

Maître Capell... @francois-seguin : un avis ?

(Et pour éviter que ça pique les yeux de certain(e)s, on pourrait simplement écrire "Toutes ces offres seront publiées". ;)

Reply:
Merci pour cette question !
[Robert](https://dictionnaire.lerobert.com/guide/accord-du-verbe-avec-un-sujet-collectif) et ~la Reine~ [le Roi ](https://www.noslangues-ourlanguages.gc.ca/fr/cles-de-la-redaction/ensemble-accord-du-verbe)donnent la même règle: _ensemble_ est un nom collectif dont l'utilisation avec un article défini commande l'usage du singulier.
On écrira donc:
```suggestion
                  L’ensemble de ces offres sera publié une fois votre
                  structure validée.
```